lib/ios/RNNStackPresenter.m

```diff
@@ -62,7 +62,7 @@ - (void)applyOptions:(RNNNavigationOptions *)options {
     [_topBarPresenter applyOptions:withDefault.topBar];
     
     [stack setNavigationBarBlur:[withDefault.topBar.background.blur getWithDefaultValue:NO]];
-    [stack setNavigationBarLargeTitleVisible:[withDefault.topBar.largeTitle.visible getWithDefaultValue:NO]];
```

Comment:
How come we don't need to pass a value to `setNavigationBarLargeTitleVisible` anymore?